drivers/ma_yya_report/app/controllers/ma_yya_report/warehouse_reports/reports_controller.rb

```diff
@@ -0,0 +1,74 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module MaYyaReport::WarehouseReports
+  class ReportsController < ApplicationController
+    include WarehouseReportAuthorization
+    before_action :set_report, only: [:show, :destroy, :details]
+
+    def index
+      @pagy, @reports = pagy(report_scope)
+      @filter = ::Filters::FilterBase.new(user_id: current_user.id)
+    end
+
+    def create
+      @filter = ::Filters::FilterBase.new(user_id: current_user.id).set_from_params(filter_params)
```

Comment:
I'd like to transition to `update`
```suggestion
      @filter = ::Filters::FilterBase.new(user_id: current_user.id).update(filter_params)
```